portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,42 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+/**
+ * Fetches a random quote from the server and adds it to the DOM.
+ */
+function getRandomQuote() {
+  console.log('Fetching a random quote.');
```

Comment:
Client side log is in general not very useful for an external facing product(because it only logs in client's browser and never send to the developers). Unless you want to make the debugging easier, you don't have to add it.

Reply:
Ok sounds good, thank you!